sklearn/utils/extmath.py

```diff
@@ -243,6 +240,11 @@ def randomized_range_finder(
     # Sample the range of A using by linear projection of Q
     # Extract an orthonormal basis
     Q, _ = linalg.qr(safe_sparse_dot(A, Q), mode="economic")
```

Comment:
Hmmm so this changes now the dot product between A and Q will use float64 right, so will use twice more memory.

This seems a too wide-reaching change only to make the benchmark run ...

Reply:
Ups I completely missed that we changed something `extmath.py`.
Sorry for merging this one.

We need to revert it.

Reply:
Sorry for this bad PR. @lesteve @glemaitre. I have overlooked the downsides of using the f64 matrix. Do you have other ideas on how to fix the benchmark?

Reply:
No worries, things like that happen! Thanks a lot for this PR, we are now very close to have a working benchmark :wink: